src/main/java/io/apiman/cli/command/AbstractCommand.java

```diff
@@ -16,24 +16,26 @@
 
 package io.apiman.cli.command;
 
-import com.google.common.collect.Maps;
+import static io.apiman.cli.util.AuthUtil.DEFAULT_SERVER_PASSWORD;
```

Comment:
Sorry, looks like my eclipse being annoying and reordering dependencies. If you have settings, I'll use those instead.